### DISCUSSION TITLE
[Prettier Plugin]  Remove spaces

### DISCUSSION THREAD

**Discussion by MayconTrix** [tailwindlabs/tailwindcss#13519] [Ideas]
My suggestion would be to add functionality to remove spaces at the beginning at the end and double spaces in the middle.

**Comment by alex-krasikau** [comment]
Hey @MayconTrix!

This behavior is going to break the code for some developers who use string concatenation. Consider the following example:
```js
<div className={"p-8 " + (primary ? "bg-black" : "bg-gray")}>
```

This example produces the `p-8 bg-black` result. If the plugin were to remove the space at the end, this would lead to an incorrect class name: `p-8bg-black`.

**Reply by MayconTrix** [reply]
You're right, I hadn't thought about this point. thanks.
